src/clims/models/file.py

```diff
@@ -16,13 +17,21 @@ def as_excel(self):
         """
         Returns the file as an excel file
         """
-        return load_workbook(self.file.getfile())
+        with NamedTemporaryFile(suffix='.xlsx') as tmp:
+            with open(tmp.name, 'wb') as f:
+                for _, chunk in enumerate(self.file.getfile()):
+                    f.write(chunk)
+
+            workbook = load_workbook(tmp.name, data_only=True)
+
+        return workbook
 
     def as_xml(self):
         raise NotImplementedError()
 
     def as_csv(self):
-        from clims.services import Csv
+        # TODO: Add @lru_cache to the method when ready
```

Comment:
I'm note sure I see the purpose of setting a `lru_cache` here. What was your reasoning?

Reply:
It's something that remained from a pair session with Steinar. In beginning as_csv was decorated with @lru_cache, and it caused problems. Then we moved it to a TODO :)